scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -26,40 +31,50 @@ task WithdrawSamples {
     String project_id
     String dataset_name
 
-    Array[String] sample_names
+    String sample_name_column_name_in_file
+    File sample_names_to_include_file
+    # should be in the format "2022-01-01 00:00:00 UTC"
+    String withdrawn_timestamp
   }
 
   meta {
-    description: "Withdraw Samples from GVS by marking them as 'withdrawn' in the sample_info table"
+    description: "Given a list of samples for a callset, withdraw samples from GVS that are not included by marking them as 'withdrawn' in the sample_info table with a passed in timestamp."
     # Might not be strictly necessary to make this volatile, but just in case:
     volatile: true
   }
 
   command <<<
-    set -e
-    set -x
+    set -o errexit -o nounset -o xtrace -o pipefail
+    echo "project_id = ~{project_id}" > ~/.bigqueryrc
+
+    # get just the sample_name values from sample_names_to_include_file based on the
+    # sample_name_column_name_in_file into sample_names.tsv
+    col_num=$(head -n1 ~{sample_names_to_include_file} | tr '\t' '\n' | grep -Fxn ~{sample_name_column_name_in_file} | cut -f1 -d:)
```

Comment:
This line could maybe use some more docs, these are some pretty tricky transformations

Reply:
also might want quotes around this in case the magic column name has spaces
```suggestion
    col_num=$(head -n1 ~{sample_names_to_include_file} | tr '\t' '\n' | grep -Fxn '~{sample_name_column_name_in_file}' | cut -f1 -d:)
```